quicksort.java

```diff
@@ -0,0 +1,66 @@
+public class MyQuickSort {
```

Comment:
This class must implements the ISortingAlgorithm interface

src/main/java/sortBench/algorithms/ISortingAlgorithm.java